tests/mentorship_relation/test_dao_creation.py

```diff
@@ -219,6 +219,25 @@ def test_dao_create_mentorship_relation_with_mentor_already_in_relation(self):
 
         self.assertTrue(1, len(query_mentorship_relations))
 
+    def test_dao_create_mentorship_relation_with_good_args_but_invalid_timestamp(self):
+        dao = MentorshipRelationDAO()
+        data = dict(
+            mentor_id=self.first_user.id,
+            mentee_id=self.second_user.id,
+            end_date=1580338800000000,
+            notes=self.notes_example,
+            tasks_list=TasksListModel()
+        )
+
+        # Use DAO to create a mentorship relation
+
+        result = dao.create_mentorship_relation(self.first_user.id, data)
+
+        print(result)
```

Comment:
Can you remove this print, it was used for testing, but it does not need to be here right?

Reply:
Sure, forgot to remove!

Reply:
Removed :D